### PR TITLE
Add EIP: Sharable interface for Borrowing

### PR DESCRIPTION
**Pull request by briankostar**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by YaroslavPysanskyi** [comment]
My name is Yaroslav Pysanskyi. My UID:F311097

File: EIPS/eip-6882.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+     * @param account The address of the delegatee
+     */
+    function isAllowed(address owner, address account) external view returns (bool);
+}
+
+```
+
+## Rationale
+
+The rationale behind creating a standard API for borrowing tokens is to improve the user and developer experience when interacting with lending protocols. By providing a borrowing aggregator, users can access aggregated data from multiple lending protocols in a single place and easily move funds between them to achieve the best borrowing rates.
```
Comment (SamWilsn) [inline EIPS/eip-6882.md:96]: This paragraph belongs in the Motivation section. The Rationale section should justify individual technical choices made within the proposal itself (eg. why `withdraw` returns the final amount withdrawn), while the Motivation section justifies the proposal as a whole.

File: EIPS/eip-6882.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+## Backwards Compatibility
+
+This proposal is designed to be compatible with major lending protocols, such as Aave V2/V3 and Compound V3. However, it may not support some protocols like Compound V2 and Morpho, due to significant differences in their interfaces and the lack of support for 'credit delegation' parameters.
+
+## Test Cases
+
+Test cases for the proposed EIP will be provided in a well-written test template for contracts that comply with the standard.
```
Comment (SamWilsn) [inline EIPS/eip-6882.md:104]:
```suggestion
```

You can omit this section for now, and put it back when you add your tests cases to the `../assets/eip-6882/` directory.

File: EIPS/eip-6882.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+## Security Considerations
+
+By providing a well-written test template for contracts that comply with the standard, the security of the contracts can be improved, leading to better user experience and higher overall security.
```
Comment (SamWilsn) [inline EIPS/eip-6882.md:108]:
```suggestion
<!-- TODO -->
```

The paragraph currently here doesn't really provide any new information or considerations that isn't already well-known in the smart contract community. If you have specific considerations, do list them here.

**Comment by SamWilsn** [comment]
I am closing this pull request because we are in the process of separating EIPs and ERCs into distinct repositories. Unfortunately, as far as we are aware, GitHub does not provide any tools to ease this migration, so every pull request will need to be re-opened manually.

As this is a PR to create / modify an ERC, I will kindly ask you to redirect this to the new repository at [ethereum/ERCs](https://github.com/ethereum/ERCs). **We have prepared [a guide](https://github.com/ethereum/ERCs/issues/8) to help with the process.**

If there is relevant history here, please link to this PR from the new pull request.

On behalf of the EIP Editors, I apologize for this inconvenience.

**Pull request closed without merging**